vmhost/vmhooks/baseOps.go

```diff
@@ -841,6 +841,18 @@ func TransferValueExecuteWithTypedArgs(
 		}
 	}
 
+	data := ""
+	if contractCallInput != nil {
+		data = makeCrossShardCallFromInput(contractCallInput.Function, contractCallInput.Arguments)
+	}
+
+	lastRound := host.Blockchain().LastRound()
+	if host.IsBuiltinFunctionCall([]byte(data)) &&
+		lastRound >= uint64(host.EnableEpochsHandler().GetActivationEpoch(vmhost.CheckBuiltInCallOnTransferValueAndFailExecutionFlag)) {
```

Comment:
does this compare rounds with epochs?

Reply:
it compares the data coming from the enableepochsHandler -> and it uses as round.

Will refactor in another PR.